src/components/input/__tests__/input-test.js

```diff
@@ -0,0 +1,15 @@
+/** @jsx React.DOM */
+
+jest.dontMock('../code/views/inputView.jsx');
+
+var React = require('react/addons');
+var InputComponent = require('../code/views/inputView.jsx');
+var TestUtils = React.addons.TestUtils;
+
+describe('InputComponent', function() {
```

Comment:
I'm seeing a bunch of login in the template, but not many tests. Can you add tests around the logic you have within the component please?